uint8 x
uint8 y
float32 yaw
string serial_id